client/checkout/classic/giropay.js

```diff
@@ -0,0 +1,198 @@
+/* global jQuery */
```

Comment:
We removed cliend/checkout/classic/sepa.js and put the sepa specific code into index.js along with the CC payment method code. I think we should do something similar with this giropay.js file. As it is several functions such as showError are duplicated.